### DISCUSSION TITLE
Turkish language support

### DISCUSSION THREAD

**Discussion by yemreeee** [community/community#77011] [Mobile] [answered]
Please add Turkish language to the user interface. Thanks.

**Comment by HenSquared** [comment, marked as answer]
@yemreeee ,
Thanks for the feedback. After shipping support for German and Korean we recently found similar requests for other languages. Internally I've created an issue to help us track language support requests.

**Comment by MuhammetAkkan** [comment]
I use paid github Copilot. Ide's language is **English but I want copilot to give Turkish answers** in the _messaging section._ We cannot use Copilot effectively. Please solve this.

**Comment by MKTRaiYAPaiZEKAi** [comment]
🇹🇷
